sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -42,23 +42,28 @@
     private Protos.ExecutorInfo.Builder executorBuilder;
     private final PodInstance podInstance;
     private final Map<String, Map<String, String>> lastTaskEnvs;
+    private final boolean useDefaultExecutor;
 
     public PodInfoBuilder(
             PodInstanceRequirement podInstanceRequirement,
             String serviceName,
             UUID targetConfigId,
             SchedulerFlags schedulerFlags,
-            Collection<Protos.TaskInfo> currentPodTasks)
-                    throws InvalidRequirementException {
+            Collection<Protos.TaskInfo> currentPodTasks,
+            Protos.FrameworkID frameworkID,
+            boolean useDefaultExecutor) throws InvalidRequirementException {
         PodInstance podInstance = podInstanceRequirement.getPodInstance();
+        this.useDefaultExecutor = useDefaultExecutor;
```

Comment:
I don't really understand the value of making these functions static at all, since they do in fact depend on an aspect of object state, so that's why I elected to make them not-static. I'll make that uniform.